hugo/layouts/papers/single.html

```diff
@@ -26,6 +26,7 @@
   <meta property="og:title" content="{{ $paper.title }}" />
   {{ with $paper.thumbnail }}
   <meta property="og:image" content="{{ . }}" />
+  <meta property="og:image:alt" content="First page of paper PDF." />
```

Comment:
Do you think you can tailor this to the specific paper, either putting the title or the Anthology ID?

Reply:
That should be apparent from the other tags. This just describes what the image adds.